src/core/thermostat.cpp

```diff
@@ -134,6 +151,47 @@ void thermo_init_npt_isotropic() {
 }
 #endif
 
+/** Brownian thermostat initializer.
+ *
+ *  Default particle mass is assumed to be unitary in these global parameters.
+ */
+void thermo_init_brownian() {
+  /** The heat velocity dispersion corresponds to the Gaussian noise only,
+   *  which is only valid for the BD. Just a square root of kT, see (10.2.17)
+   *  and comments in 2 paragraphs afterwards, @cite Pottier2010.
+   */
+  brownian.sigma_vel = sqrt(temperature);
+  /** The random walk position dispersion is defined by the second eq. (14.38)
+   *  of @cite Schlick2010. Its time interval factor will be added in the
+   *  Brownian Dynamics functions. Its square root is the standard deviation.
+   */
+  if (temperature > 0.0) {
+    brownian.sigma_pos_inv = sqrt(brownian.gamma / (2.0 * temperature));
+  } else {
+    brownian.sigma_pos_inv =
+        brownian.gammatype_nan; // just an indication of the infinity
+  }
+#ifdef ROTATION
+  /** Note: the BD thermostat assigns the brownian viscous parameters as well.
+   *  They correspond to the friction tensor Z from the eq. (14.31) of
+   *  @cite Schlick2010.
+   */
+  /* If gamma_rotation is not set explicitly,
+     we use the translational one. */
+  if (brownian.gamma_rotation < GammaType{}) {
+    brownian.gamma_rotation = brownian.gamma;
+  }
+  brownian.sigma_vel_rotation = sqrt(temperature);
+  if (temperature > 0.0) {
+    brownian.sigma_pos_rotation_inv =
+        sqrt(brownian.gamma / (2.0 * temperature));
```

Comment:
Are we intentionally using `gamma` instead of `gamma_rotation` in the expression of `sigma_pos_rotation_inv`?

Reply:
looks like a typo, thanks, should be rotational.